doc/md_src_files/082_relatedSystems.md

```diff
@@ -105,3 +105,74 @@ In that sense, it is compatible with any of methods mentioned above or SSVC.
 However, SSVC would be better suited to address vPrioritizer's broad spectrum asset management data.
 For example, vPrioritizer aims to collect data points on topics such as asset significance.
 Asset significance could be expressed through the SSVC decision points of  [*Mission Impact*](#mission-impact) and situated [*Well-being Impact*](#well-being-impact), but it does not have a ready expression in CVSS, EPSS, or VPR.
+
+
+## SSVC using Current Information Sources
+
+Some SSVC decision points can be informed or answered by currently available information feeds or sources.
+These include two aspects of [*Exploitation*](#exploitation), [*System Exposure*](#system-exposure), [*Technical Impact*](#technical-impact), and [*Public Safety Impact*](#public-safety-impact).
+This section provides an overview of some options; we cannot claim it is exhaustive.
+Each decision point has a subsection for `Gathering Information About` it.
+These sections provide suggestions that would also contribute to creating or honing information feeds. 
+However, if there is a viable information source we have not captured, please create an [issue on the SSVC GitHub page](https://github.com/CERTCC/SSVC/issues) explaining the information source and what decision point it informs.  
+
+Various vendors provide paid feeds of vulnerabilities that are currently exploited by attacker groups.
+Any of these could be used to indicate that [*active*](#exploitation) is true for a vulnerability.
+Although the lists are all different, we expect they are all valid information sources to be used for this purpose.
+Although we are not aware of a comparative study of the different lists of active exploits, we expect they have similar properties to block lists of network touchpoints [@metcalf2015blocklist] and malware [@kuhrer2014paint].
+Namely, each list has a different view and vantage on the problem, which makes them appear to be different, but each list does a good job at accurately representing its particular vantage at a point in time.
+
+
+[*System Exposure*](#system-exposure) should be informed by the various scanning platforms such as and Shodan and Shadowserver.
+A service on a device should be scored as [*open*](#system-exposure) if such a general purpose Internet scan finds that the service responds.
+Such scans do not find all [*open*](#system-exposure) systems, but any system they find should be considered [*open*](#system-exposure).
+Scanning software, such as the open-source Nessus, could be used to scan for connectivity inside an organization to catalogue what devices should be scored [*controlled*](#system-exposure) if, say, the scan finds them on an internal network where devices regularly connect to the Internet.  
+
+
+There are some information sources that are not exactly suited to make SSVC decisions, but with a bit of work could be used directly.
```

Comment:
If they can be "used directly with a bit of work" then they are "suited" for it. I think "were not designed with SSVC in mind, but can be adapted to work" is the idea you're getting at here.